pnpm-workspace.yaml

```diff
@@ -0,0 +1,65 @@
+packages:
+  - xwiki-platform-core/xwiki-platform-blocknote/xwiki-platform-blocknote-webjar/src/main/node/
+  - xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-webjar/src/main/node/
+  - xwiki-platform-core/xwiki-platform-notifications/xwiki-platform-notifications-webjar/src/main/node/
+catalogMode: strict
+cleanupUnusedCatalogs: true
```

Comment:
Dependencies are automatically removed from this list when all packages stop using them.